vendor/hbi/hbi-mla20-3l0606c.yaml

```diff
@@ -1,4 +1,4 @@
-name: HBI emergency lighting series 300 eagle
```

Comment:
Please make sure the name starts with a capital.

Probably easiest is to find in files with the right filter (`vendor/**/*.yaml`), case sensitive and regex enabled for `name: [a-z]`

Reply:
And what about this for example?
https://github.com/TheThingsNetwork/lorawan-devices/blob/8cd4bbf6dc318e23f1c6ec1cfa89e8ba42704537/vendor/tektelic/t0006923-edoctor.yaml#L1

The device is officially called `eDoctor` with a small e and capital D. Do we just ignore their naming convention or?

Reply:
Yes those are fine